config/routes.rb

```diff
@@ -206,6 +206,12 @@
 
     namespace :web do
       resource :settings, only: [:update]
+      resource :push_subscriptions, only: [:create] do
+        # Why doesn't PUT /api/web/push_subscriptions/:id work?
```

Comment:
"resource" means there's no index, show with no ID (e.g. there is always one item)

"resources" is what gives you both index and show with ID